install-server.sh

```diff
@@ -84,6 +84,8 @@ install_k3s() {
 
 install_helm() {
     echo "Installing Helm..."
+    # FIXME: why does /usr/local/bin vanish from PATH?
```

Comment:
This actually happened to me too, I really don't know and I think it's worth investigating in a second moment